# Joint State Controller - publishes joint states
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position controllers for each joint
front_right_shoulder_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_shoulder
  pid: {p: 100.0, i: 0.01, d: 10.0}

front_right_leg_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_leg
  pid: {p: 100.0, i: 0.01, d: 10.0}

front_right_foot_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_right_foot
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_right_shoulder_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_right_shoulder
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_right_leg_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_right_leg
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_right_foot_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_right_foot
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_left_shoulder_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_left_shoulder
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_left_leg_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_left_leg
  pid: {p: 100.0, i: 0.01, d: 10.0}

rear_left_foot_position_controller:
  type: effort_controllers/JointPositionController
  joint: rear_left_foot
  pid: {p: 100.0, i: 0.01, d: 10.0}

front_left_shoulder_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_shoulder
  pid: {p: 100.0, i: 0.01, d: 10.0}

front_left_leg_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_leg
  pid: {p: 100.0, i: 0.01, d: 10.0}

front_left_foot_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_left_foot
  pid: {p: 100.0, i: 0.01, d: 10.0}
